week-03/main.js

```diff
@@ -25,12 +39,22 @@ var list = [1, 2, 3, 4, 5];
 
 // sayHello('Mer') // "Hello Mer!"
 
+function sayHello (name) {
+  return "hello" + name + "!";
+};
+
 // Define a function `createRange` that takes `min` and `max` as parameters and returns an array with all the numbers
 // between min and max
 // Example:
 
 // createRange(4, 8) // [4, 5, 6, 7, 8]
 
+function createRange (min, max) {
+  for(var i = min; i < max; i++) {
+    return([i]);
```

Comment:
`return` breaks out of the function, therefore you don't want to return until you've created a whole array. How would you add things to an array and then when you're done return that array?